### PR TITLE
[CS-1392] Upgrade walletconnect version to 1.5, update pod version in gemfile

### PR DESCRIPTION
**Pull request by topwebtek7**

### CODE REVIEW

**Review by DominicSherman — COMMENTED** [review]
Should we kick off a release from this branch to ensure everything is fully functional before merging?

**Comment by topwebtek7** [comment]
> Should we kick off a release from this branch to ensure everything is fully functional before merging?

I think that's good idea.. @kdowlin thoughts?

**Comment by DominicSherman** [comment]
> > Should we kick off a release from this branch to ensure everything is fully functional before merging?
> 
> I think that's good idea.. @kdowlin thoughts?

I'll kick one off now 👍

File: ios/Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "fastlane"
-gem 'cocoapods', '1.9.1'
+gem 'cocoapods', '1.10.1'
```
Comment (danibonilha) [inline ios/Gemfile:4]: Let's not forget to rebase and re-run `bundle exec pod install` before merging

**Review by lukemelia — COMMENTED** [review]
It seems to me that these should have been two separate PRs: one for walletconnect upgrade and one for the cocoapods version update.

**Comment by topwebtek7** [comment]
close this pr as opened another #279

**Pull request closed without merging**